homeassistant/components/homekit/manifest.json

```diff
@@ -3,5 +3,5 @@
   "name": "HomeKit",
   "documentation": "https://www.home-assistant.io/integrations/homekit",
   "requirements": ["HAP-python==2.8.2"],
-  "codeowners": []
+  "codeowners": ["@bdraco"]
```

Comment:
:+1:

Reply:
Thanks for looking.  I figured I'm answering them anyways and I'd like to know if I broke anything so I can fix it :)